Content.Shared/Chemistry/Reagent/ReagentPrototype.cs

```diff
@@ -102,6 +102,12 @@ public sealed partial class ReagentPrototype : IPrototype, IInheritingPrototype
         [DataField]
         public float Viscosity;
 
+         /// <summary>
+        /// Should this reagent work on the dead?
+        /// </summary>
+        [DataField("worksOnTheDead")]
+        public bool WorksOnTheDead = false;
```

Comment:
```suggestion
        [DataField]
        public bool WorksOnTheDead;
```

datafields don't need the name in the attribute + false is the default value for booleans so you can leave it off.